nekoyume/Assets/_Scripts/State/RxProps.Arena.cs

```diff
@@ -108,6 +102,27 @@ public static
             IReadOnlyAsyncUpdatableRxProp<(ArenaInformation current, ArenaInformation next)>
             ArenaInfoTuple => _arenaInfoTuple;
 
+        private static readonly ReactiveProperty<(
+            int currentTicketCount,
+            int maxTicketCount,
+            int progressedBlockRange,
+            int totalBlockRange,
+            string remainTimespanToReset)> _arenaTicketProgress =
```

Comment:
In my humble opinion, Why don't you make this tuple to `struct`?

Reply:
I just wanted to reduce the worry of where to define the struct.😅